Sources/Apodini/SyntaxTreeVisitor/ContextNode.swift

```diff
@@ -88,13 +88,12 @@ class ContextNode {
             // we need to check if `OptionalContextKey` is type of `ContextKey`
             // aka if the `ContextKey provides a default value. Because if it provides a defaultValue
             // we need to call `reduce(...)` with it before inserting.
-            let visitor = StandardContextKeyTypeVisitor()
-            let defaultValue = visitor(contextKey)
 
             // if defaultValue is nil the contextKey didn't conform to `ContextKey` => doesn't have a default value
-            if let defaultValue = defaultValue {
+            if let contextKey = contextKey as? HasDefaultValue.Type {
```

Comment:
# Context
`AssociatedTypeRequirementsTypeVisitor` is still broken for release builds. So this is just moving away from it, since we can